tests/qchem/of_tests/test_convert.py

```diff
@@ -987,15 +999,134 @@ def test_excited_configurations_error(excitation):
+                (6, 5): 0.003359774446779245,
+                (10, 9): 0.003359774446779244,
+                (18, 17): 0.003359774446779245,
+                (5, 6): 0.003359774446779244,
+                (5, 5): -0.18938190575578503,
+                (9, 10): 0.003359774446779243,
+                (9, 9): -0.18938190575578523,
+                (17, 18): 0.003359774446779244,
+                (17, 17): -0.18938190575578503,
+            },
+        ),
+    ],
+)
+def test_rcisd_state(molecule, basis, symm, tol, wf_ref):
+    r"""Test that _rcisd_state returns the correct wavefunction."""
+
+    mol = pyscf.gto.M(atom=molecule, basis=basis, symmetry=symm)
+    myhf = pyscf.scf.RHF(mol).run()
+    myci = pyscf.ci.CISD(myhf).run()
+
+    wf_cisd = qchem.convert._rcisd_state(myci, tol=tol)
 
-    mol = pyscf.gto.M(atom=[["H", (0, 0, 0)], ["H", (0, 0, 0.71)]], basis="sto6g")
+    assert wf_cisd.keys() == wf_ref.keys()
+    assert np.allclose(abs(np.array(list(wf_cisd.values()))), abs(np.array(list(wf_ref.values()))))
+
+
+@pytest.mark.parametrize(
+    ("molecule", "basis", "symm", "tol", "wf_ref"),
+    [
+        (
+            [["H", (0, 0, 0)], ["H", (0, 0, 0.71)]],
+            "sto6g",
+            "d2h",
+            1e-1,
+            {(1, 1): 0.9942969030671576, (2, 2): -0.10664740292693174},
+        ),
+        (
+            [["Li", (0, 0, 0)], ["Li", (0, 0, 0.71)]],
+            "sto6g",
+            "d2h",
+            1e-1,
+            {
+                (7, 7): 0.8886970081919591,
+                (11, 11): -0.3058459002168582,
+                (19, 19): -0.30584590021685887,
+                (35, 35): -0.14507552387854625,
+            },
+        ),
+    ],
+)
+def test_uccsd_state(molecule, basis, symm, tol, wf_ref):
+    r"""Test that _uccsd_state returns the correct wavefunction."""
+
+    mol = pyscf.gto.M(atom=molecule, basis=basis, symmetry=symm)
     myhf = pyscf.scf.UHF(mol).run()
-    myci = pyscf.ci.UCISD(myhf).run()
+    mycc = pyscf.cc.UCCSD(myhf).run()
 
-    with monkeypatch.context() as m:
-        m.setitem(sys.modules, "pyscf", None)
+    wf_ccsd = qchem.convert._uccsd_state(mycc, tol=tol)
+
+    assert wf_ccsd.keys() == wf_ref.keys()
+    assert np.allclose(abs(np.array(list(wf_ccsd.values()))), abs(np.array(list(wf_ref.values()))))
+
+
+@pytest.mark.parametrize(
+    ("molecule", "basis", "symm", "tol", "wf_ref"),
+    [
+        (
+            [["H", (0, 0, 0)], ["H", (0, 0, 0.71)]],
+            "sto6g",
+            "d2h",
+            1e-1,
+            {(1, 1): 0.9942969030671576, (2, 2): -0.10664740292693242},
+        ),
+        (
+            [["Li", (0, 0, 0)], ["Li", (0, 0, 0.71)]],
+            "sto6g",
+            "d2h",
+            1e-1,
+            {
+                (7, 7): 0.8886969878256522,
+                (11, 11): -0.30584590248164206,
+                (19, 19): -0.30584590248164145,
+                (35, 35): -0.14507552651170982,
+            },
+        ),
+    ],
```

Comment:
These input parameters are duplicated for both the uccsd and rccsd tests, consider defining and storing them once at the beginning of the test file and calling them where required.

Reply:
Done.